src/main/java/net/earthcomputer/clientcommands/command/AliasCommand.java

```diff
@@ -38,10 +38,24 @@
     private static final DynamicCommandExceptionType NOT_FOUND_EXCEPTION = new DynamicCommandExceptionType(arg -> new TranslatableText("commands.calias.notFound", arg));
 
     private static HashMap<String, String> aliasMap = new HashMap<>();
+    private static CommandDispatcher<ServerCommandSource> dispatcher;
 
     static {
         try {
+            ClientPlayNetworkHandler networkHandler = MinecraftClient.getInstance().getNetworkHandler();
+            assert networkHandler != null;
+            dispatcher = (CommandDispatcher<ServerCommandSource>) (CommandDispatcher<?>) networkHandler.getCommandDispatcher();
+
             getAliases();
+            if (!aliasMap.isEmpty()) {
```

Comment:
Are you sure this `if`-statement is needed?

Reply:
ah, probably not, good call